src/scrape_up/IPL/ipl.py

```diff
@@ -0,0 +1,37 @@
+import requests
+
+# Replace 'YOUR_API_KEY' with your actual NewsAPI.org API key
+API_KEY = 'YOUR_API_KEY'
+NEWS_API_ENDPOINT = 'https://newsapi.org/v2/top-headlines'
+IPL_QUERY = 'IPL'
+NEWS_API_PARAMS = {
+    'q': IPL_QUERY,
+    'language': 'en',
+    'apiKey': API_KEY
+}
+
+def fetch_ipl_news():
+    try:
+        response = requests.get(NEWS_API_ENDPOINT, params=NEWS_API_PARAMS)
+        response.raise_for_status()
+        news_data = response.json()
+        return news_data['articles']
+    except requests.exceptions.RequestException as e:
+        print(f"Error occurred while fetching IPL news: {e}")
+        return None
+
+def display_ipl_news(news_articles):
```

Comment:
We follow Classes to build modules and methods. Refactor your code accordingly.

Check other modules like `github` and see how they are implemented. Don't just blindly push your changes.